client/components/CandidateReview/CandidateTestPlanRun/InstructionsRenderer.jsx

```diff
@@ -169,19 +170,128 @@ const InstructionsRenderer = ({
         commandsContent
     );
 
-    const assertions = [...pageContent.instructions.assertions.assertions];
-    const assertionsContent = parseListContent(assertions);
-
+    headingLevel = 5;
     const Heading = `h${headingLevel}`;
 
     return (
         <>
             <NumberedList>{allInstructionsContent}</NumberedList>
-            <Heading>{pageContent.instructions.assertions.header}</Heading>
-            {pageContent.instructions.assertions.description}
-            <NumberedList>{assertionsContent}</NumberedList>
             {settingsContent.length ? settingsContent : null}
 
+            {renderableContent.commands.map(({ id, settings }, i) => {
```

Comment:
A key thing missing here is that [assertionExceptions](https://github.com/w3c/aria-at/wiki/Test-Format-Definition-V2#assertionexceptions) aren't being handled here.

To confirm this, I did the following:
* There is an open PR which should add a test plan version with `assertionExceptions`, https://github.com/w3c/aria-at/pull/1015.
* I set the import script locally to `tests/Radio-activedescendant-refactor` and ran the script
* Navigated to Radio Group Example Using aria-activedescendant Test Plan V24.02.19 from the Data Management page
* Confirmed the commands shown for Test 1 across all 3 ATs didn't match the [built preview link of the aria-at PR](https://deploy-preview-1015--aria-at.netlify.app/review/radiogroup-aria-activedescendant).

`assertionExceptions` are available on the `renderableContent.commands` value being mapped through.